flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1,4 +1,8 @@
 {
+    "one.ablaze.floorp": {
+        "finish-args-unnecessary-xdg-data-applications-create-access": "Needed for manually creating application .desktop files for Floorp's Web Apps feature",
+        "finish-args-unnecessary-xdg-data-icons-create-access": "Needed as Floorp's Web Apps feature downloads and installs icons for each website that the user adds",
```

Comment:
```suggestion
        "finish-args-unnecessary-xdg-data-icons-create-access": "Needed as Floorp's Web Apps feature downloads and installs icons for each website that the user adds"
```

Reply:
oh wait whoops didn't catch that, mb late night mistake too lol